src/pages/validator/external-chains/centrifuge.mdx

```diff
@@ -12,22 +12,42 @@ Set up your Centrifuge Mainnet or Testnet node.
 - [Releases](https://github.com/centrifuge/centrifuge-chain/releases)
 - Docker OR `rustup` is installed
 
-## Steps
+### Optional: Using a snapshot from the Centrifuge dev team
+-> This will make your node pretty much ready to receive requests in within hours rather than days.
+
+  Using Centrifuge provided snapshots (faster)
+    ```
+    gsutil ls gs://centrifuge-snapshots/{mainnet,testnet-demo}
+    # Download the bundle
+    gsutil cp gs://centrifuge-snapshots/mainnet/YYYY-MM-DD-snap.tar.lz4 $DATA_FOLDER_PATH/
+    lz4 -d -c -q  $DATA_FOLDER_PATH/YYYY-MM-DD-snap.tar.lz4 | tar -xvf - -C /path/to/destination
+    ```
+   Inspect the /path/to/destination, it should contain a 'chain' and a 'relay-chain' directory, the parachain and relay chain 
```

Comment:
`/path/to/destination`, `'chain'`, `'relay-chain'`

Reply:
changed for more clear variables and added ` instead of '